quarkus/runtime/src/test/java/org/keycloak/quarkus/runtime/configuration/ConfigurationTest.java

```diff
@@ -62,6 +62,9 @@ public void testCamelCase() {
         initConfig();
         String value = Config.scope("camelCaseScope").get("camelCaseProp");
         assertEquals(value, "foobar");
+
+        value = Config.scope().root().get("spi-camelCaseScope-camelCaseProp");
+        assertEquals(value, "foobar");
```

Comment:
nitpicking, intellij shows the warning `Arguments to 'assertEquals()' in wrong order`